src/classes/RundownManager.ts

```diff
@@ -83,8 +83,8 @@ export class RundownManager {
 				if (rawSegment) {
 					const segment: UnrankedSegment = {
 						externalId: rawSegment.identifier,
-						name: rawSegment.fields.title || '',
-						modified: ParseDateFromInews(rawSegment.fields.modifyDate),
+						name: rawSegment.fields.title ? rawSegment.fields.title : '',
```

Comment:
A suggestion is to use
```suggestion
    name: rawSegment.fields.title ?? ''
```
, to enforce when we use the fallback, as well as making the expression more compact. The current solution will use the fallback for any falsy value, including an empty string, which, I guess, is a valid input. The suggestion will only use the fallback if encountering `null` or `undefined`.
This might also apply to the next line ("modified" attribute), if an empty string is a valid input for `ParseDataFromInews`.